indent/ruby.vim

```diff
@@ -549,6 +569,14 @@ function! s:AfterIndentKeyword(pline_info)
     " level
     if s:Match(info.plnum, s:end_end_regex)
       let ind = indent('.')
+    elseif s:IsAssignment(info.pline, col)
+      if g:ruby_indent_end_alignment == 'hanging'
+        " hanging indent
+        let ind = col + info.sw - 1
+      else
+        " align with variable
+        let ind = indent(info.plnum) + info.sw
+      endif
```

Comment:
If this is commented out, https://github.com/vim-ruby/vim-ruby/pull/329/files#diff-42baa6ece1da9b1f054f57486a47c523R116 fails.

```
        variable = if something == something_else
          something_else
        elsif other == none
          none
        else
          other
        end
```